prowler/providers/aws/services/eventbridge/eventbridge_bus_cross_account_access/eventbridge_bus_cross_account_access.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "aws",
+  "CheckID": "eventbridge_bus_cross_account_access",
+  "CheckTitle": "Ensure that AWS EventBridge event buses do not allow unknown cross-account access for delivery of events.",
+  "CheckType": [],
+  "ServiceName": "eventbridge",
+  "SubServiceName": "eventbus",
+  "ResourceIdTemplate": "arn:partition:events:region:account-id:event-bus/resource-id",
+  "Severity": "high",
+  "ResourceType": "AwsEventsEventbus",
+  "Description": "Ensure that AWS EventBridge event buses do not allow unknown cross-account access for delivery of events.",
+  "Risk": "If an AWS EventBridge event bus allows unknown cross-account access for delivery of events, it can lead to unauthorized access to the event bus and its events.",
+  "RelatedUrl": "https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CWE_GettingStarted.html",
```

Comment:
Please include a more specific link here.

Reply:
The more specific link is in the Recommendation part.